fauna/query_builder.py

```diff
@@ -104,32 +77,15 @@ def __init__(self, fragments: Optional[List[Fragment]] = None):
         self._rendered = None
```

Comment:
Does `fragments` really need to be Optional in the `__init__` method? If it's required, then this `QueryBuilder` subclass would look even more like a `Fragment`.

Reply:
I removed append for now, but I think it's fine to keep this optional. It gives us greater flexibility with how we use these builders. e.g. we can add append back and init and append afterward, which maybe we (or someone) will want depending on how they're building their queries.